### PR TITLE
Use Rotational to detect HDD disk for LocalVolumeSet for Vmware LSO deployment

### PR DESCRIPTION
**Pull request by prsurve**
Fixes: #3098 

Signed-off-by: Pratik Surve <prsurve@redhat.com>

### CODE REVIEW

**Review by wusui — DISMISSED** [review]
Code looks good.  You may want to expand on the comment.  Something lie:

Since we don't have datastore with SSD on our current VMware machines, localvolumeset doesn't detect NonRotational disk.  As a workaround we are setting Rotational to device MechanicalProperties to detect HDD disk

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -1090,6 +1090,11 @@ def setup_local_storage(storageclass):
             constants.LOCAL_VOLUME_SET_YAML
         )
 
+        # as a workaround we are setting Rotational to deviceMechanicalProperties to detect HDD disk
```
Comment (ebenahar) [inline ocs_ci/deployment/deployment.py:1093]: do we have an issue to track the W/A?
also, do we need to set this value everytime?

Comment (prsurve) [reply ocs_ci/deployment/deployment.py:1094]: i have created  an issue here #3098, yes need to set that value of all the deployment of VMware LSO

**Comment by prsurve** [comment]
> Code looks good. You may want to expand on the comment. Something lie:
> 
> Since we don't have datastore with SSD on our current VMware machines, localvolumeset doesn't detect NonRotational disk. As a workaround we are setting Rotational to device MechanicalProperties to detect HDD disk

Done

**Review by ramkiperiy — APPROVED** [review]

**Review by sidhant-agrawal — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**